browser/ui/views/frame/brave_browser_view.cc

```diff
@@ -130,6 +131,24 @@ class ContentsBackground : public views::View {
 BEGIN_METADATA(ContentsBackground)
 END_METADATA
 
+// A `ContentsWebView` that activates its contents when it gets focus.
+class ActivatableContentsWebView : public ContentsWebView {
+  METADATA_HEADER(ActivatableContentsWebView, ContentsWebView)
+ public:
+  using ContentsWebView::ContentsWebView;
+  ~ActivatableContentsWebView() override = default;
+
+  // ContentsWebView:
+  void OnFocus() override {
+    ContentsWebView::OnFocus();
+    if (web_contents()) {
+      web_contents()->GetDelegate()->ActivateContents(web_contents());
```

Comment:
Curious why this need to be called explicitely because `ContentWebView::OnFocus()` doesn't do this for original page view.

Reply:
My theory is that they just don't assume page can be focused without/before activated. But in our case, secondary web view can be focused while it's inactive.

Reply:
Then, maybe we also have to handle its devtools webview in the same way?

Reply:
Well, I'm not sure about that. When devtools are undocked and the target tab is inactive, activating devtools window doesn't activate the tab.

Reply:
Tested with docked mode, and got below crash with docked mode.
Also, that tab is not activated when I clicked devtools area

Reply:
> Also, that tab is not activated when I clicked devtools area

Yes, that's what I expected. because focusing devtools doesn't activate inactive tab in non split view.

Reply:
Ah, I see. it's upstream behavior.
I think we need to discuss about this in the separated issue because clicking devtools area also means split view area. It's weird that it doesn't get focused when devtools area.